TTS/tts/models/overflow.py

```diff
@@ -200,15 +200,15 @@ def _format_aux_input(self, aux_input: Dict, default_input_dict):
             }
         )
         if aux_input:
-            return format_aux_input(aux_input, default_input_dict)
-        return None
+            return format_aux_input(default_input_dict, aux_input.copy())
+        return default_input_dict.copy()
 
     @torch.no_grad()
     def inference(
         self,
         text: torch.Tensor,
-        aux_input={"x_lengths": None, "sampling_temp": None, "max_sampling_time": None, "duration_threshold": None},
-    ):  # pylint: disable=dangerous-default-value
```

Comment:
I would prefer having it so people are aware of what auxiliary input the mode supports during inference.